app/services/quarterly_message_employee_matcher.rb

```diff
@@ -0,0 +1,52 @@
+require "business_time"
+
+class QuarterlyMessageEmployeeMatcher
```

Comment:
@jessieay yeah the other class screams for renaming just as you said.  I just wasn't sure how many fingerprints I was allowed to leave on the code so I tried not to go overboard.